client/src/components/common/Header/index.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import "./style.css";
+
+const Header = () => {
+  return (
+    <div className="header">
+      <div className="header_logo">
+        <img
+          className="header_logo-img"
+          src="https://svgshare.com/i/Jru.svg"
+          alt="GSG Logo"
+        />
+        <div className="header_logo-title">
+          Events <span className="header_logo-subtitle">Booker</span>
+        </div>
+      </div>
+      <div>
+        <h3 className="header_title">
+          Welcome to <span className="header_subTitle">GSG event app</span>
+        </h3>
+        <h4 className="sub-header">
+          See the available events at GSG and book your favorite
```

Comment:
Probably not necessary here but if you define all of your strings in a separate file and import them into the component, you'll find it much easier if you want to add different languages to the app later